docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -1,4 +1,6 @@
 ---
+aliases:
+  - ../tutorial/play-with-grafana-mimir
```

Comment:
I don't think we need any aliases because the content has only ever lived at https://grafana.com/tutorials/play-with-grafana-mimir/ as far as I can tell.

Reply:
I guess you could put in 
```suggestion
  - /tutorials/play-with-grafana-mimir/
```

Preemptively in case we ever remove the mount of the tutorial.